### PR TITLE
Initial Code Structure with Navigation Drawer

### PR DESCRIPTION
**Pull request by jatinluthra14**

### CODE REVIEW

**Comment by jatinluthra14** [comment]
Done

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    defaultConfig {
+        applicationId "org.jbossoutreach.contact_groups_server"
+        minSdkVersion 27
```
Comment (garg-anuj) [inline app/build.gradle]: minSdkVersion should never be so high

Comment (jatinluthra14) [reply app/build.gradle]: Is 24 okay?

**Comment by garg-anuj** [comment]
Thanks for work @jatinluthra14

**Pull request closed without merging**